pkg/frontend/frontend_test.go

```diff
@@ -4,178 +4,14 @@ package frontend
 // Licensed under the Apache License 2.0.
 
 import (
-	"bytes"
-	"encoding/json"
-	"errors"
-	"net/http"
-	"net/http/httptest"
-	"reflect"
 	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/gorilla/mux"
 	"github.com/sirupsen/logrus"
-	"github.com/sirupsen/logrus/hooks/test"
-	kerrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"github.com/Azure/ARO-RP/pkg/api"
 )
 
-func TestAdminReply(t *testing.T) {
```

Comment:
worth salvaging any of this for a TestReply() ?

Reply:
can do